src/main/java/page_objects/LoginPage.java

```diff
@@ -0,0 +1,32 @@
+package page_objects;
+
+import elements.Button;
+import elements.Label;
+import elements.TextBox;
+import helpers.ElementHelper;
+import org.openqa.selenium.By;
+
+public class LoginPage extends BasePage {
+    private final TextBox txtEmail = new TextBox(By.id("username"));
+    private final TextBox txtPassword = new TextBox(By.id("password"));
+    private final Button btnLogin = new Button(By.cssSelector(".form-actions>input[title='Login']"));
+    private final Label msgFinalErrorMsg = new Label(By.cssSelector("p.message"));
+    private final Label msgEmailErrorMsg = new Label(By.cssSelector(".username>.validation-error"));
+    private final Label msgPasswordErrorMsg = new Label(By.cssSelector(".password>.validation-error"));
+    private final Label lblLoginTitle = new Label(By.cssSelector("#content>h1"));
```

Comment:
Add more attribute for h1 tag

Reply:
I added an attribute for it and because it appears on some pages so I decided to put it on BasePage. Please check it out for me. Thank you.

Reply:
I have just rechecked for this locator, there are no attribute for h1 tag, so it is ok to use #content>h1

Reply:
Thank you. :)